Makefile

```
@@ -19,3 +23,20 @@ markdown-toc:
 			echo markdown-toc: no TOC markers, skipping $$f; \
 		fi; \
 	done
+
+.PHONY: markdown-link-check
+markdown-link-check:
+	@if ! npm ls markdown-link-check; then npm install; fi
```

```suggestion
	@if ! npm ls markdown-link-check; then npm ci; fi
```

I generally prefer using `npm ci`. Same with other 2 cases.

---

I copied the checks from https://github.com/open-telemetry/opentelemetry-specification :-)

Will change to `ci`.